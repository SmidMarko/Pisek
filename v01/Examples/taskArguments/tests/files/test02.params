{"executionArgs": "-51 7"}
