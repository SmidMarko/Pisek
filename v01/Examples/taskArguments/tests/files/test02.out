-51 7.00
